client/src/pages/news_notice/AnnouncementTable.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { FunctionComponent, useState, useEffect } from 'react';
+import { withTheme } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
+import styled from 'styled-components';
+import { scroller } from 'react-scroll';
+import map from 'lodash/map';
+import { useHistory } from 'react-router';
+
+import { Table } from '../../components/table/Table';
+import { Announcement } from '../../types/announcement';
+
+import { AnnouncmentDetail } from './AnnouncmentDetail';
+
+interface AnnouncementProps {
+  className?: string;
+  announcements?: Announcement[];
+  id: string;
+}
+
+const tableColumns = [
+  { title: 'Title', field: 'title' },
+  { title: 'Date Posted', field: 'createdAt' },
+];
+
+/**
+ * Announcement Table Component.
+ */
+const UnstyledAnnouncementsTable: FunctionComponent<AnnouncementProps> = ({
```

Comment:
`AnnouncementsTable` does not really represent the component - it also displays the selected announcement